diff --git a/gdb/configure b/gdb/configure
index e8979f0321..ccadeef1af 100755
--- a/gdb/configure
+++ b/gdb/configure
@@ -9101,7 +9101,7 @@ fi
 
 
 
-try_guile_versions="guile-2.2 guile-2.0"
+try_guile_versions="guile-2.0"
 have_libguile=no
 case "${with_guile}" in
 no)
diff --git a/gdb/configure.ac b/gdb/configure.ac
index a40860ad43..544a8a1244 100644
--- a/gdb/configure.ac
+++ b/gdb/configure.ac
@@ -1135,7 +1135,7 @@ AC_MSG_RESULT([$with_guile])
 dnl We check guile with pkg-config.
 AC_PATH_PROG(pkg_config_prog_path, pkg-config, missing)
 
-try_guile_versions="guile-2.2 guile-2.0"
+try_guile_versions="guile-2.0"
 have_libguile=no
 case "${with_guile}" in
 no)
